### ISSUE TITLE
Fail to load stylesheets

### ISSUE THREAD

**Issue by tylergoodman** [OmniSharp/omnisharp-atom#509]
The styles in this package depend on variables that are provided by the atom theme that is being used. 
https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L1
Some of these variables are not provided or otherwise not available anymore and this is causing the package to fail to load.  Specifically, the `@base-accent-color` variable is no longer here, see:
https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L504
https://github.com/atom/atom-dark-ui/blob/master/styles/ui-variables.less

**Atom Version**: 1.0.11
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.24.7

### Stack Trace

Failed to load the omnisharp-atom package

```
At variable @base-accent-color is undefined in C:\Users\t\.atom\packages\omnisharp-atom\styles\omnisharp-atom.less:504:21

LessError: variable @base-accent-color is undefined
  at C:\Users\t\.atom\packages\omnisharp-atom\styles\omnisharp-atom.less:504:21
```

**Comment by david-driscoll** [comment]
This should hopefully be fixed with v0.24.9 (I'm not getting the error myself sadly)

**Comment by brad-jones** [comment]
Slightly different issue but I believe it might be a side effect of https://github.com/OmniSharp/omnisharp-atom/commit/d9f92702813601c5bf71eecb1db0ae941e6bd223, 
LESS gives slightly deceptive line numbers in it's exceptions.

I just upgraded to v0.24.9 and I now get:
```
LessError: variable @ui-lightness is undefined
  at C:\Users\Brad\.atom\packages\omnisharp-atom\styles\omnisharp-atom.less:504:8
```

If we look at [line 504](https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L504) we note that there is no reference to ```@ui-lightness``` at all.
It is actually used in the above mixin at [line 488](https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L488).

I just commented out the section like so:
```less
/*& when (@ui-lightness > 50%) {
    border-color: transparent; // hide border on light backgrounds
}*/
```

And now omnisharp-atom loads correctly.

**Comment by david-driscoll** [comment]
What two themes do you have selected?

On Tue, Sep 15, 2015 at 11:53 PM Brad Jones <notifications@github.com>
wrote:

> Slightly different issue but I believe it might be a side effect of
> d9f9270
> <https://github.com/OmniSharp/omnisharp-atom/commit/d9f92702813601c5bf71eecb1db0ae941e6bd223>,
>
> LESS gives slightly deceptive line numbers in it's exceptions.
>
> I just upgraded to v0.24.9 and I now get:
>
> LessError: variable @ui-lightness is undefined
>   at C:\Users\Brad\.atom\packages\omnisharp-atom\styles\omnisharp-atom.less:504:8
>
> If we look at line 504
> <https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L504>
> we note that there is no reference to @ui-lightness at all.
> It is actually used in the above mixin at line 488
> <https://github.com/OmniSharp/omnisharp-atom/blob/master/styles/omnisharp-atom.less#L488>
> .
>
> I just commented out the section like so:
>
> /*& when (@ui-lightness > 50%) {    border-color: transparent; // hide border on light backgrounds}*/
>
> And now omnisharp-atom loads correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OmniSharp/omnisharp-atom/issues/509#issuecomment-140618718>
> .
>
-- 
Thanks,
David

**Comment by brad-jones** [comment]
UI Theme: Atom Dark
Syntax Theme: Monokai Soda

**Comment by willl** [comment]
Interesting, it works fine for me when using the `One Dark` UI Theme with Monokai.

But as soon as I switch over to `Atom Dark`, I get the same `@ui-lightness` issue that @brad-jones reported.

**Comment by peterblazejewicz** [comment]
Gusy, it seems  `ui-lightness` is fixed with 0.24.10 (Atom Dark tested, OS X)

**Comment by mat-mcloughlin** [comment]
Yeah 0.24.10 contains the fix
